core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -151,6 +156,8 @@
         // Set the primitive properties.
         this.workerGroup = requireNonNull(workerGroup, "workerGroup");
         this.shutdownWorkerGroupOnStop = shutdownWorkerGroupOnStop;
+        this.serviceWorkerGroup = serviceWorkerGroup;
+        this.shutdownServiceWorkerGroupOnStop = shutdownServiceWorkerGroupOnStop;
```

Comment:
We don't need these because this is virtual host and service level property, as @ikhoon said above.